### PR TITLE
Added basic job tests

### PR DESCRIPTION
**Pull request by mikedao**
Created basic specs for job model.

### CODE REVIEW

File: spec/models/job_spec.rb [hunk 1]
```diff
@@ -1,5 +1,33 @@
...
+  it "has a description" do
+    job = create(:job)
+
+    expect(job.description.class).to eq(String)
+  end
+
+  it "has a location" do
+    job = create(:job)
+
+    expect(job.description).to eq("Clean stuff")
```
Comment (EmilyMB) [inline spec/models/job_spec.rb:19]: Duplicate, except this line 25, which should be on line 20, no?

File: spec/factories.rb [hunk 2]
```diff
@@ -1,2 +1,8 @@
 FactoryGirl.define do
```
Comment (EmilyMB) [inline spec/factories.rb:1]: Probably better to use Nathan's factory here since it will pass his validations (in particular...belonging to company).

Comment (mikedao) [inline spec/factories.rb:1]: If Nathan's already done this, we can just delete the branch.

**Comment by mikedao** [comment]
Merged in Nathan's work, set up relationships in factory.

**Comment by nathanows** [comment]
:dog: :poodle: :dog2: say this is all clear, its good with me.

**Comment by AdamSmith910** [comment]
all clear

**Pull request merged**